src/common/framework/fixture.ts

```diff
@@ -271,6 +285,11 @@ export class Fixture<S extends SubcaseBatchState = SubcaseBatchState> {
         this.rec.expectationFailed(new Error('threw unexpectedly' + m));
       } else {
         this.expectErrorValue(expectedError, ex, new Error(m));
+        if (checkForStackProperty) {
+          if (!(ex instanceof Error && typeof ex.stack === 'string')) {
+            this.rec.expectationFailed(new Error('missing stack' + m));
```

Comment:
nit. maybe `'threw as expected, but missing stack' + m` to be consistent with `shouldReject`?